.github/workflows/run-tests.yml

```diff
@@ -60,9 +60,14 @@ jobs:
 
   style_checker:
     runs-on: ubuntu-latest
-    container: citus/stylechecker:latest
+    container: citus/stylechecker:no-py
```

Comment:
I wanted to rebuild the docker image, but failed as some of the python package versions in the container no longer is available in Python Package Index. I ended up removing all of those broken dependencies in https://github.com/citusdata/the-process/pull/156 to create a new docker image that has working `citus_indent` tool.